dashboard/app/views/plc/course_units/show.html.haml

```diff
@@ -0,0 +1,75 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/levelbuilder', media: 'all'
+  = javascript_include_tag 'js/plc'
+
+%p#notice= notice
+
+%p
+  %b Plc course:
+  = link_to @course_unit.plc_course.name, @course_unit.plc_course
+%p
+  %b Unit name:
+  = @course_unit.unit_name
+%p
+  %b Unit description:
+  = @course_unit.unit_description
+%p
+  %b Unit order:
+  = @course_unit.unit_order
+
+%p
+  Evaluation Questions
+
+%p
```

Comment:
The piece for creating new questions and answers was brought over from plc/course/show